src/utilities/buildASTSchema.js

```diff
@@ -466,6 +452,42 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
 }
 
 /**
+ * This takes the ast of a schema document produced by the parse function in
+ * src/language/parser.js.
+ *
+ * If no schema definition is provided, then it will look for types named Query
+ * and Mutation.
+ *
+ * Given that AST it constructs a GraphQLSchema. The resulting schema
+ * has no resolve methods, so execution will use default resolvers.
+ */
+export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
+  const config: any = buildASTSchemaComponents(ast);
```

Comment:
Shouldn't need an `:any` cast here, at least not without clear comment as to why. Seems unsound